.github/workflows/webdriver-test.yml

```diff
@@ -0,0 +1,25 @@
+name: Webdriver Test
+
+on: [push, pull_request]
+
+jobs:
+    build:
+      runs-on: ubuntu-latest
+
+      steps:
+        - uses: actions/checkout@v2
+
+        - name: Build Docker images
+          run: docker-compose build
+
+        - name: Run Docker containers
+          run: |
+            docker-compose up -d
+            docker-compose run --rm wdio
+        
+        - name: Archive screenshots
```

Comment:
Do we have something similar to public https://github.com/oncokb/oncokb-public/blob/master/.github/workflows/screenshot-test.yml#L20-L31 to compare latest/base?

Reply:
We do. I save the entire screenshots folder which contains the image that was just taken, the baseline image, and the diff